src/main/resources/application-release.properties

```diff
@@ -10,12 +10,11 @@ cloud.aws.region.static=eu-central-1
 cloud.s3.bucket-name=untitled-testing-system
 
 # Email notifications
-email-notifications.enabled=true
+email-notifications.enabled=false
 email-notifications.from-email=untitled.noreply@gmail.com
 
 email-notifications.template-path.test-assigned=test_assigned.ftlh
-email-notifications.template-path.test-deassigned=test_assigned.ftlh
-email-notifications.template-path.test-started=test_assigned.ftlh
-email-notifications.template-path.test-verified=test_assigned.ftlh
-email-notifications.template-path.coach-assigned=test_assigned.ftlh
-email-notifications.template-path.coach-deassigned=test_assigned.ftlh
+email-notifications.template-path.test-deassigned=test_deassigned.ftlh
+email-notifications.template-path.test-verified=test_verified.ftlh
+email-notifications.template-path.coach-assigned=coach_assigned.ftlh
+email-notifications.template-path.coach-deassigned=coach_deassigned.ftlh
```

Comment:
Let's also add the template for expired tests

Reply:
Oh, here we go again